### DISCUSSION TITLE
Is the Form Builder tool from the website open source?

### DISCUSSION THREAD

**Discussion by JoeRoddy** [react-hook-form/react-hook-form#10347] [Q&A] [answered]
Hi!  We need to build a form creation tool for our users, just wondering if the code for the form builder is open sourced somewhere.

For reference:
https://react-hook-form.com/form-builder/

**Comment by Moshyfawn** [comment]
#8791

**Comment by JoeRoddy** [comment, marked as answer]
The actual source code, for anyone else whose looking:
https://github.com/react-hook-form/documentation/blob/master/src/components/BuilderPage.tsx
